lib/winapi/winbase.h

```diff
@@ -25,43 +24,44 @@ extern "C"
 // CreateThread or _beginthreadex, stacks are always committed memory in the Xbox
 #define STACK_SIZE_PARAM_IS_A_RESERVATION 0x00010000
 
-#define THREAD_PRIORITY_ABOVE_NORMAL 1
-#define THREAD_PRIOIRTY_BELOW_NORMAL (-1)
-#define THREAD_PRIORITY_HIGHEST 2
-#define THREAD_PRIORITY_IDLE (-15)
-#define THREAD_PRIORITY_LOWEST (-2)
-#define THREAD_PRIORITY_NORMAL 0
+#define THREAD_PRIORITY_ABOVE_NORMAL  1
+#define THREAD_PRIOIRTY_BELOW_NORMAL  (-1)
+#define THREAD_PRIORITY_HIGHEST       2
+#define THREAD_PRIORITY_IDLE          (-15)
+#define THREAD_PRIORITY_LOWEST        (-2)
+#define THREAD_PRIORITY_NORMAL        0
 #define THREAD_PRIORITY_TIME_CRITICAL 15
-#define THREAD_PRIORITY_ERROR_RETURN 0x7FFFFFFF
+#define THREAD_PRIORITY_ERROR_RETURN  0x7FFFFFFF
 
-typedef VOID (WINAPI *PFLS_CALLBACK_FUNCTION)(PVOID);
-#define FLS_OUT_OF_INDEXES 0xFFFFFFFF
+typedef VOID(WINAPI *PFLS_CALLBACK_FUNCTION)(PVOID);
+#define FLS_OUT_OF_INDEXES    0xFFFFFFFF
 #define FLS_MAXIMUM_AVAILABLE 64
-#define TLS_OUT_OF_INDEXES FLS_OUT_OF_INDEXES
+#define TLS_OUT_OF_INDEXES    FLS_OUT_OF_INDEXES
 #define TLS_MINIMUM_AVAILABLE FLS_MAXIMUM_AVAILABLE
 
 typedef struct _INIT_ONCE
 {
     PVOID Ptr;
 } INIT_ONCE, *PINIT_ONCE, *LPINIT_ONCE;
-#define INIT_ONCE_STATIC_INIT {0}
+#define INIT_ONCE_STATIC_INIT       {0}
 #define INIT_ONCE_CTX_RESERVED_BITS 2
-#define INIT_ONCE_CHECK_ONLY 0x00000001UL
-#define INIT_ONCE_ASYNC 0x00000002UL
-#define INIT_ONCE_INIT_FAILED 0x00000004UL
-typedef BOOL (CALLBACK *PINIT_ONCE_FN) (PINIT_ONCE, PVOID, PVOID *);
+#define INIT_ONCE_CHECK_ONLY        0x00000001UL
+#define INIT_ONCE_ASYNC             0x00000002UL
+#define INIT_ONCE_INIT_FAILED       0x00000004UL
+typedef BOOL(CALLBACK *PINIT_ONCE_FN)(PINIT_ONCE, PVOID, PVOID *);
```

Comment:
Does removing the white-spaces here not change the behavior?

Reply:
That can be the case with `#define`s, where there must not be a whitespace between the name and parameter list for function-style macros, but typedef isn't sensitive to whitespace changes.

Reply:
Well, in that case I have no qualms for a merge.